src/RFC-0312_DANHighLevelSpecification.md

```diff
@@ -498,15 +507,17 @@ to penalties and stake slashing if enabled within the contract specification.
 ### Changes to the constitution
 [contract constitution change]: #changes-to-the-constitution
 
-Any changes to the [contract constitution] MUST happen at checkpoints. This also refers to any changes to the VNC.
+Any changes to the [contract constitution] can happen at any time. This also refers to any changes to the VNC. Only the [CC] may 
```

Comment:
`[constitution amendment]` needs to be linked to the description of the process.
```suggestion
Changes to the [contract constitution] can happen at any time through the [constitution amendment] process. This also refers to any changes to the VNC. Only the [CC] may 
```